src/components/calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './Calculator.css';
```

Comment:
- [ ] This import does not work because the file is called `calculator.css`, but you are trying to import it as `Calculator.css`.  Kindly correct that and the import will work! 💪